esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -516,7 +516,7 @@ bool CWsDfuEx::onDFUSpace(IEspContext &context, IEspDFUSpaceRequest & req, IEspD
         else
         {
             Owned<IEspSpaceItem> item64 = createSpaceItem();
-            if (stricmp(countby, COUNTBY_OWNER))
+            if (stricmp(countby, COUNTBY_OWNER))  // MORE: this looks like it should be strieq
```

Comment:
@wangkx please can you check this

Reply:
The logic here is: if countby is not COUNTBY_OWNER. You may change it to !strieq.